src/components/Features/Faq/Faq.module.scss

```diff
@@ -0,0 +1,87 @@
+@import '../../../index';
+
+$color: $--color-white;
+$link-color: $--color-gamma;
+$link-color-hover: $--color-gamma-1;
+$visited-link-color: $--color-alpha-6;
+$visited-link-color-hover: $--color-alpha-6-hover;
+$border-color: $--color-alpha-3;
+$background-answer: $--color-alpha-3;
+
+.faq {
+  color: $color;
+  max-height: 500px;
+  overflow-y: auto;
+  text-align: left;
+
+  a {
+    color: $link-color;
+    &:hover {
+      color: $link-color-hover;
+    }
+    &:visited {
+      color: $visited-link-color;
+
+      &:hover {
+        color: $visited-link-color-hover;
+      }
+    }
+  }
+
+  .container {
+    color: $color;
+    padding: 13px;
+    border-radius: 7px;
+    border-color: $border-color;
+    border-style: solid;
+    border-width: 2px;
+    margin: 10px 0 0;
+    overflow-y: auto;
+    max-height: 372px;
+
+    > ol {
+      margin-top: -20px;
+    }
+  }
+
+  ol {
+    line-height: 1.6;
+    padding-inline-start: 20px;
+    margin-block-start: 0;
+    padding-right: 16px;
+
+    .question {
+      position: relative;
+      margin-bottom: 40px;
+
+      h4 {
+        letter-spacing: 0.7px;
+      }
+
+      &::marker {
+        font-size: 16px;
+        font-weight: 600;
+      }
+
+      & * {
+        position: relative;
+        z-index: 1;
+      }
```

Comment:
For what is this?

Reply:
to lift the text above the light-purple background that's behind the answers